db/schema.rb

```diff
@@ -21,7 +21,7 @@
     t.bigint "record_id", null: false
     t.bigint "blob_id", null: false
     t.datetime "created_at", null: false
-    t.index ["blob_id"], name: "index_active_storage_attachments_on_blob_id"
+    t.index ["blob_id"], name: "ix_active_storage_attachments_on_blob_id"
```

Comment:
huh how come  all this stuff is changing ?
Seems its happening to all the indexes